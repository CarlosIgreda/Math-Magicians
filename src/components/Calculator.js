import React from 'react';
import '../styles/App.css';
import Buttons from './Buttons';

function Calculator() {
  return (
    <div className="calc">
      <Buttons />
    </div>
  );
}
export default Calculator;
