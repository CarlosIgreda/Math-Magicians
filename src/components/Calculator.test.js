import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Calculator from './Calculator';

describe('Calculator', () => {
  it('should render correctly', () => {
    const { container } = render(
      <MemoryRouter>
        <Calculator />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
