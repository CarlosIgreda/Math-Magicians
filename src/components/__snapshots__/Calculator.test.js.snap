// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator should render correctly 1`] = `
<div>
  <div>
    <nav
      class="header"
    >
      <h1
        class="title"
      >
        üìê Math Magiciansüìè
      </h1>
      <ul
        class="menu"
      >
        <li
          class="menu-item"
        >
          <a
            href="/Math-Magicians"
          >
            Home
          </a>
        </li>
        <li
          class="menu-item"
        >
          <a
            href="/Math-Magicians/calculator"
          >
            Calculator
          </a>
        </li>
        <li
          class="menu-item"
        >
          <a
            href="/Math-Magicians/quote"
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
    <div
      class="calc"
    >
      <p
        class="math-text"
      >
        Let's do some math!
      </p>
      <div
        class="buttons"
      >
        <div
          class="display"
        >
          0
        </div>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          AC
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          +/-
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          %
        </button>
        <button
          class="button math-symbols"
          type="button"
        >
          √∑
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          7
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          8
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          9
        </button>
        <button
          class="button math-symbols"
          type="button"
        >
          x
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          4
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          5
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          6
        </button>
        <button
          class="button math-symbols"
          type="button"
        >
          -
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          1
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          2
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          3
        </button>
        <button
          class="button math-symbols"
          type="button"
        >
          +
        </button>
        <button
          class="button numbers-and-extras number0"
          type="button"
        >
          0
        </button>
        <button
          class="button numbers-and-extras"
          type="button"
        >
          .
        </button>
        <button
          class="button math-symbols"
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
